src/index.ts

```diff
@@ -113,32 +113,33 @@ export = (app: Application) => {
         }
       }
     `, {
-      owner: owner,
-      repo: repo,
-      refQualifiedName: `refs/heads/${branchName}`
+      owner,
+      repo
     }, {})
     if (!result.data) { return [] }
-    return result.data.repository.ref.associatedPullRequests.nodes.map((node: any) => ({
-      number: node.number,
-      repo: node.repository.name,
-      owner: node.repository.owner.login
-    }))
+    const allPullRequests = result.data.repository.pullRequests.edges.map((edge: any) => {
+      const node = edge.node
+      return {
+        number: node.number,
+        commitId: node.headRefOid,
+        repo: node.repository.name,
+        owner: node.repository.owner.login
+      }
+    })
+
+    return allPullRequests.filter((pr: any) => pr.commitId === commitId)
   }
 
   app.on([
     'status'
   ], async context => {
-    const branches = context.payload.branches as { name: string }[]
-    const validBranches = branches.filter(branch => branch.name !== 'master')
-    const pullRequestResponses = await Promise.all(validBranches.map(branch =>
-      getAssociatedPullRequests(context.github, {
-        owner: context.payload.repository.owner.login,
-        repo: context.payload.repository.name,
-        branchName: branch.name
-      })
-    ))
-
-    const pullRequests = flatten(pullRequestResponses)
+    const pullRequests = await
+    getAssociatedPullRequests(context.github, {
```

Comment:
```suggestion
    const pullRequests = await getAssociatedPullRequests(context.github, {
```
Consistent formatting.